spec/models/cider_spec.rb

```diff
@@ -2,4 +2,10 @@
 
 RSpec.describe Cider, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
+
+  it "is valid with valid attributes"
```

Comment:
I would probably omit these behaviors and  include them in a separate pull request